.github/workflows/deploy-customer.yml

```
@@ -91,19 +57,4 @@ jobs:
-          aws ec2 revoke-security-group-ingress \
-            --group-id ${{ secrets.SECURITY_GROUP_ID }} \
-            --protocol tcp \
-            --port 22 \
-            --cidr ${{ steps.ip.outputs.ipv4 }}/32 || echo "Rule may not exist or already removed"
-
-      # 배포 결과 확인
-      - name: Verify deployment
-        run: |
-          echo "🎉 Deployment completed!"
\ No newline at end of file
+            nohup java -jar /home/ec2-user/eatngo-customer-api/*.jar --spring.profiles.active=dev > app.log 2>&1 &
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)